packages/cozy-harvest-lib/src/helpers/konnectors.js

```diff
@@ -103,6 +109,109 @@ export const needsFolder = konnector => {
   )
 }
 
+/**
+ * Base directories are directory where konnector may copy their data.
+ * They are expressed as variables which then need to be localized.
+ * Default is `$administrative`.
+ */
+const allowedBaseDirVariables = ['$administrative', '$photos']
+
+/**
+ * Parse base directory, based on given folders object.
+ * For example, it will parse `$administrative` to the given value passed in
+ * folders object. We expect to find in folders a localized value.
+ * @param  {String} baseDir base directory variable, expects `$administrative`
+ * or `$photos`
+ * @param  {Object} folders Object indexing base directory variable with
+ * corresponding localized name.
+ * @return {String}         Localized directory
+ */
+const parseBaseDir = (baseDir, folders = {}) => {
+  if (!allowedBaseDirVariables.includes(baseDir))
+    throw new Error(
+      `baseDir may only have the following values: ${allowedBaseDirVariables.join(
+        ', '
+      )})`
+    )
+
+  // Look for variable name into folders but without $ prefix
+  const parsedBaseDir = folders[baseDir.slice(1)] || DEFAULT_LOCALIZED_BAR_DIR
+  // Trim `/` and avoid multiple `/` characters with regexp
+  return trim(parsedBaseDir.replace(/(\/+)/g, '/'), '/')
+}
+
+/**
+ * Parse the given folder path using the given `variables` object.
+ * Variables are injected in `buildFolderPath` function. Available variable are:
```

Comment:
"Variables are injected in `buildFolderPath` function.", as this is an implementation detail, I would leave this out of the comment of the functions.

```suggestion
 * Available variables are:
```

Also, this is not a `parse`, it's a `render` (like for example Handlebars.render)